pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.9.RELEASE</version>
+        <version>2.6.6</version>
```

Comment:
Whitesource also reports some vulnarbilities in:
mongock-spring ( org.apache.ant:ant:jar:1.10.3]-> fix 1.10.8 and  com.google.guava:guava:jar:15.0]-> fix in  24.1.1-jre). Ideally we should migrate to some other library. But at least these vulnerabilities should be addressed. You can inspire yourself from core loader.

Reply:
@AndreiOla not completely following. `mongock-spring` is aligned with core loader.   `org.apache.ant` and `com.google.guava` are not dependencies for flow loader.